curriculum/schema/challenge-schema.js

```diff
@@ -183,7 +182,6 @@ const schema = Joi.object().keys({
     then: Joi.string()
   }),
   challengeFiles: Joi.array().items(fileJoi),
-  guideUrl: Joi.string().uri({ scheme: 'https' }),
```

Comment:
There are still references to this in the code. Specifically `prop-types.ts`. Is there a reason to leave it there?